app.js

```diff
@@ -43,23 +43,29 @@ const courseSchema = new mongoose.Schema({
 	author: String,
 	authorUrl: String,
 	courseUrl: String,
-	review: String,
 	price: Number,
 	isFree: Boolean,
-	reviewTitle: String,
-	reviewDetails: String
 });
 
 const Course = mongoose.model("Course", courseSchema);
 
+const reviewSchema = new mongoose.Schema({
```

Comment:
No need to state "id" as a field, it is automatically generated :)

Reply:
Oh cool, thanks for telling me!